myTest.java

```diff
@@ -0,0 +1,177 @@
+import com.google.gson.Gson;
+import org.junit.jupiter.api.*;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import parser.Parser;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)

```

Comment:
Let's remove Order annotation from all tests and this annotation too. Order is used in tests in exceptional cases. You must avoid use of Order annotation as much as possible.

Reply:
done